pkg/kubeclient/kubelet_client.go

```diff
@@ -133,6 +133,9 @@ func (client *KubeletClient) GetSummary(host string) (*stats.Summary, error) {
 		if entryPresent {
 			glog.V(2).Infof("unable to retrieve machine[%s] summary: %v. Using cached value", host, err)
 			entry.used = true
+			if entry.statsSummary == nil {
+				return nil, err
```

Comment:
let's add another log message here to indicate that cache is used ,but summary is nil. Otherwise we can't distinguish the case whether we populate the summary from the cache or not.